pkg/gen/python-templates/yaml.py.mustache

```diff
@@ -64,11 +64,7 @@ class ConfigFile(pulumi.ComponentResource):
         else:
             text = _read_file(file_id)
 
-        if opts is not None:
-            _opts = deepcopy(opts)
-            _opts.parent = self
-        else:
-            _opts = pulumi.ResourceOptions(parent=self)
+        _opts = pulumi.ResourceOptions.merge(opts, pulumi.ResourceOptions(parent=self))
```

Comment:
Might want to rename this to `opts` for consistency (or vice versa).

Reply:
yup yup